CHANGELOG.md

```diff
@@ -427,6 +427,7 @@ const config = {
 * File format: generates Realms with format v22 (reads and upgrades file format v5 or later for non-synced Realm, upgrades file format v10 or later for synced Realms).
 
 ### Internal
+* Upgrade `ReactTestApp` to RN 0.66.3 and to use the metro bundler to resolve links to relam and test code.
 * Add `.clang-format` file for formatting C++ code, formatted all existing code and added a lint check for C++.
```

Comment:
BTW, is the recent change to the changelog format deliberate? Older entries have the version headers as `## 10.14.0 (2022-3-24)` instead of the new underlined style. I'm asking because the update-changelog action expects the older format. Either update-changelog needs to be updated or the old format should continue to be used, depending on whether it was an intended change. Otherwise it will put the entire changelog down to 10.19.1 in every release note.

Reply:
What is the plan for the changelog format? Currently the update-changelog action https://github.com/realm/ci-actions/tree/main/update-changelog expects `## 10.20.0 (2022-8-23)` for the version headers but that's not what's being used. Is there a plan to switch to that format?

Reply:
We had not planned on that, but i'll speak with the team about this.  It may be beneficial to converge on a common format, so we can make use of this action